terratorch/datamodules/generic_pixel_wise_data_module.py

```diff
@@ -91,9 +91,9 @@ def __init__(
         test_split: Path | None = None,
         ignore_split_file_extensions: bool = True,
         allow_substring_split_file: bool = True,
-        dataset_bands: list[HLSBands | int] | None = None,
-        predict_dataset_bands: list[HLSBands | int] | None = None,
-        output_bands: list[HLSBands | int] | None = None,
+        dataset_bands: list[HLSBands | int | list[int] | str] | None = None,
+        predict_dataset_bands: list[HLSBands | int | list[int] | str ] | None = None,
+        output_bands: list[HLSBands | int | list[int] | str] | None = None,
```

Comment:
instead of `list[int]` would it work if we made it a `tuple[int, int]`? In that way we would have a check that exactly 2 elements are there

Reply:
0a84c42e0975fcc0a57740f4dc6b3340d3e956a2